### ISSUE TITLE
[EN][NPC][Northrend] Missing rares

### ISSUE THREAD

**Issue by sigmundfreud2016** [Atlantiss/BugTracker#8756]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Some rare spawns are definitely missing. I love killing rares while leveling or taming them with my hunter. The most important ones such as TLPD or spirit beasts do exist, but a lot of others never spawn. Let's take a look at Zul'Drak:
I've killed these 2 many times:
http://wowrarespawns.blogspot.de/2011/06/terror-spinner.html
http://wowrarespawns.blogspot.de/2011/07/gondria.html
however I never ever saw these 2, not even their corpses:
http://wowrarespawns.blogspot.de/2011/09/zuldrak-sentinel.html
http://wowrarespawns.blogspot.de/2011/07/griegen.html

I really doubt that somebody could farm these 2 non-stop while totally ignoring spider and kitty. That means both sentinel and griegen don't spawn at all.

**Revision**: 5277

**Comment by jissabraham** [comment]
i have killed them so many times

**Comment by TranceAccount** [comment]
http://www.wowhead.com/achievement=2257/frostbitten

**Comment by TranceAccount** [comment]
http://atlantiss.eu/armory/character/387018/achievements/14780#view-page
Credits to Pom.

**Comment by sigmundfreud2016** [comment]
@jissabraham then do it again and make a fresh screenshot. It statistically impossible that I've killed Gondria DOZENS of times, but have never seen Griegen nor Zuldrak Sentinel.

**Comment by Lebiedz** [comment]
I've killed Griegen few days ago. They are spawning, you are just unlucky. I know that feel too :)

![img_2017-01-04_11_25_19](https://cloud.githubusercontent.com/assets/13126019/21639490/a0bcfe8c-d270-11e6-93de-2999c4aabd13.png)

**Comment by sigmundfreud2016** [comment]
@Lebiedz it's not about luck, they are not world bosses after all, they should take just 6-12 hours to respawn, not days or weeks. If I killed/tamed even such mobs as Loque'nahak, Arcturus and Skoll, which are heavily camped by hunters, there should be zero problems finding Griegen at least once in ~50 tries.

**Comment by Lebiedz** [comment]
You may be right, but I'm doubtful if we as players can obtain any reliable data about this topic (because all we have is "how it should be like", and "how it could be now"). It would certainly by helpful if somebody from server staff could look into this in terms of database entries. However, I can't see it happening anytime soon :/

**Comment by sigmundfreud2016** [comment]
@Lebiedz indeed, exact spawn times are unobtainable, we can only rely on people's personal camping experience on retail and common sense.

**Comment by sigmundfreud2016** [comment]
http://wowrarespawns.blogspot.de/2011/07/seething-hate.html may be missing or spawns in wrong place, I have never seen it

**Comment by TranceAccount** [comment]
Griegen spawned today. He's dead now.

**Comment by Fabiana-B** [comment]
I saw Griegen alive.

And accordingly to @Lebiedz image, I got Vyragoza yesterday :>

**Comment by sigmundfreud2016** [comment]
@TranceAccount @Fabiana-B please read all posts before commenting, I've killed Griegen myself 5 days ago. The only question is if his (and not only his) spawn timer is normal or way too long. It took me 3 days of checking him every 2 hours. Gundrak Sentinel didn't spawn for 2nd time btw. Neither Gondria, nor Terror Spinner do NOT require such dedication.

**Comment by sigmundfreud2016** [comment]
@Fabiana-B tell me where was Vyragoza. One player has told me that he has found TLPD near camp Tunka'lo while hunting for Dirkee, which is not normal (see pic). Most guides recommend camping near temple of storms.
![image](https://cloud.githubusercontent.com/assets/12395407/21768578/8da2f9a6-d67a-11e6-9d6f-9bdf2e79060e.png)

**Comment by TranceAccount** [comment]
@sigmundfreud2016 You have never said anything about what the supposed spawn timer is supposed to be. Find proof of their spawn timers first.

**Comment by sigmundfreud2016** [comment]
@TranceAccount impossible, unless you find a former blizzard employee. As I said:
> exact blizzlike spawn times are unobtainable, we can only rely on people's personal camping experience on retail and common sense.
